BUILD

```diff
@@ -292,3 +314,19 @@ upgrading GCC. If you have libstdc++ corresponding to GCC 5.x or 6.x, you *must*
 old ABI, but Dinit will be broken if you upgrade to GCC 7. If you have libstdc++ from GCC 7, you
 *must* build with the new ABI. If the wrong ABI is used, Dinit may still run successfully but any
 attempt to load a non-existing service, for example, will cause Dinit to crash.
+
+
+Special note for Clang (on systems using Libcxxrt C++ runtime)
```

Comment:
either: _the_ LIbcxxrt C++ runtime
or: LIbcxxrt _as the_ C++ runtime

Remove the parentheses, the runtime is just as significant as the compiler for this issue.